src/MountConfig.js

```diff
@@ -24,6 +24,55 @@ class MountConfig extends SchemaDerivedConfig {
         '^/mountpoints$': NamedPairHandler('path', 'url'),
       },
     });
+    this._sanitized = null;
+  }
+
+  get mountpoints() {
+    if (!this._sanitized) {
+      this._sanitized = (this._content.mountpoints || []).map((mp) => {
+        const m = { ...mp };
+        if (!m.path.endsWith('/')) {
+          m.path += '/';
+        }
+        if (!m.type && m.url) {
+          if (!m.id && m.url.startsWith('https://drive.google.com/')) {
```

Comment:
Instead of hard-coding this here, I'd follow an approach similar to this: https://github.com/adobe/helix-embed/blob/67b263a7bac87be4c4538271a6c9e250a4546f04/src/embed.js#L82-L87

- create a list of handlers that each consist of a `matcher` (your if statement) and a `rewriter` (the body of the if statement)
- iterate over all matchers and apply the corresponding rewriter

Reply:
Maybe we should also have a `MountPointHandler` for the proxies above instead of a getter.

Reply:
> Maybe we should also have a `MountPointHandler` for the proxies above instead of a getter.

I don't understand this sentence.